From 82f60023e3d4b33624c85a149915b411b504e426 Mon Sep 17 00:00:00 2001
From: "Anthony G. Basile" <blueness@gentoo.org>
Date: Wed, 23 Jan 2013 20:16:57 -0500
Subject: [PATCH] src/libudev/util.c: fix typo introduce at commit 1ebf647

The first argument of fcntl should be 'fd', the file descriptor.

Signed-off-by: Anthony G. Basile <blueness@gentoo.org>
---
 src/libudev/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libudev/util.c b/src/libudev/util.c
index 1d57a01..d3982b9 100644
--- a/src/libudev/util.c
+++ b/src/libudev/util.c
@@ -4586,7 +4586,7 @@ int fopen_temporary(const char *path, FILE **_f, char **_temp_path) {
         fd = mkostemp(t, O_WRONLY|O_CLOEXEC);
 #else
         fd = mkstemp(t);
-        fcntl(t, F_SETFD, FD_CLOEXEC);
+        fcntl(fd, F_SETFD, FD_CLOEXEC);
 #endif
         if (fd < 0) {
                 free(t);
-- 
1.7.12.4

